### PR TITLE
New package: hoppscotch.Hoppscotch version 23.12.6.1

### PR DESCRIPTION
**Pull request by xVemu**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146354)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @xVemu,

In a Windows 10 VM, this package seems to install normally. But when run, from either the Desktop shortcut or the Start menu shortcut, a window opens and immediately closes. Is this expected?

**Comment by Trenly** [comment]
@stephengillie - It installs and runs fine in a Win11 sandbox; Perhaps `MinOSVersion` needs to be added, or perhaps there's a missing dependency which is causing the crash in Win10?

**Comment by stephengillie** [comment]
> @ stephengillie - It installs and runs fine in a Win11 sandbox; Perhaps `MinOSVersion` needs to be added, or perhaps there's a missing dependency which is causing the crash in Win10?

Yes, this would appear to be a good idea, as it also installs and launches normally in a Windows 11 VM:
image

File: manifests/h/hoppscotch/Hoppscotch/23.12.6.1/hoppscotch.Hoppscotch.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: hoppscotch.Hoppscotch
+PackageVersion: 23.12.6.1
+InstallerLocale: en-US
```
Comment (stephengillie) [inline manifests/h/hoppscotch/Hoppscotch/23.12.6.1/hoppscotch.Hoppscotch.installer.yaml:6]:
```suggestion
InstallerLocale: en-US
MinimumOSVersion: 10.0.22621.0
```

**Comment by xVemu** [comment]
It's not fully compatible with Windows 10 yet, so I agree that we should use `MinOSVersion` until https://github.com/hoppscotch/hoppscotch/issues/3524 is not fixed.

File: manifests/h/hoppscotch/Hoppscotch/23.12.6.1/hoppscotch.Hoppscotch.installer.yaml (hunk 1 above)
Comment (xVemu) [reply manifests/h/hoppscotch/Hoppscotch/23.12.6.1/hoppscotch.Hoppscotch.installer.yaml]: Somehow it breaks pipelines

**Review by stephengillie — APPROVED** [review]

**Pull request merged**